src/pages/ImportData.js

```diff
@@ -151,7 +259,21 @@ const ImportData = () => {
             ])
           }]
         ),
-        isImporting && spinnerOverlay
+        isCloneOpen && h(NewWorkspaceModal, {
+          cloneWorkspace: selectedTemplateWorkspace,
+          customCloneTitle: `Clone ${selectedTemplateWorkspace.workspace.name} and Import Data`,
+          customCloneButtonText: 'Clone and Import',
+          customMessage: noteMessage,
+          onDismiss: () => setIsCloneOpen(false),
+          onSuccess: w => {
+            setMode('template')
```

Comment:
This should set the mode to 'existing', because the clone has succeeded at this point. The idea is that if the import step fails due to a network blip, we'll leave the UI in a consistent state, and they could just click 'Import' to try again with the workspace they just created.